### DISCUSSION TITLE
Pulsar and Codeberg

### DISCUSSION THREAD

**Discussion by Crojav** [pulsar-edit/.github#285] [General]
New to pulsar, i see that there is a github intergration. But what if you like to push to codeberg insteat of github.
Is there a way to configure this?

**Comment by savetheclocktower** [comment]
Not at present.

Pulsar was forked from Atom, which was a GitHub project. The`github` package was therefore written by GitHub themselves. We maintain it and keep it from breaking, but otherwise don't touch it that much. (Insanely complicated; uses an old version of React; hard to build from source, so it's hard to contribute fixes or verify the fixes that others contribute.)

Personally, I've only ever used the _Git_ side of the `github` package; I use it to stage my commits, write commit messages, and resolve conflicts. But I ignore the _GitHub_ tab and simply push and pull manually from the command line.

**Reply by Crojav** [reply]
Thank you! for the fast and helpfull respons. 

> Personally, I've only ever used the Git side of the github package; I use it to stage my commits, write commit messages, and resolve conflicts. But I ignore the GitHub tab and simply push and pull manually from the command line.

I will follow that same strategie - have to figure it a bit out and see how this all works out for me. So any future help would be welcome.
